client/js/components/month/month-prev-button.js

```diff
@@ -1,19 +1,19 @@
 /** @jsx React.DOM */
 var React = require('react');
 var AppViewActions = require('../../actions/view-action-creator');
+var moment = require('moment');
 
 var MonthPrevButton = React.createClass({
   handleClick: function () {
-    //TEMP The 'prev' argument passed in will be changed in the real impelementation
-    AppViewActions.getMonthData('prev');
+    var date = moment(this.props.year + this.props.month, 'YYYYMMM');
+    date = date.subtract(1, 'month');
+    AppViewActions.getMonthData(date);
   },
   render: function () {
     return (
-      <div> 
-        <button className="month-prev" onClick={this.handleClick}>Prev Month</button>       
-      </div>
+      <div className="arrow arrow-left" onClick={this.handleClick}></div>
     );
   }
 });
 
-module.exports = MonthPrevButton;
\ No newline at end of file
+module.exports = MonthPrevButton;
```

Comment:
that may be something we could refactor later. for now, it may be easiest to just do something similar to the day-header, where the logic is all in one place